charts/pvault-server/templates/_helpers.tpl

```diff
@@ -154,3 +154,37 @@ Usage:
 {{- end -}}
 {{ $dst | toYaml }}
 {{- end -}}
+
+{{/*
+Validates if a key (name) exists in an ENV like object, K/V and list of env are supported.
```

Comment:
Changed approach, this is no longer needed but keeping it for the future as it might be useful.